<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMultiLine</class>
 <widget class="QDialog" name="DialogMultiLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多线绘制</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>82</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>83</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>间距</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDraw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>绘制</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Num">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Distance">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="RbHorizontalLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>水平线</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RbVerticalLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>垂直线</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RbSymbolPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>54</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>元件端点</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
